equinix/fabric_mapping_helper.go

```diff
@@ -1209,3 +1209,19 @@ func CorrelationIdWithCharset(length int, charset string) string {
 func CorrelationId(length int) string {
 	return CorrelationIdWithCharset(length, allowed_charset)
 }
+
+func additionalInfoContainsAWSSecrets(info []interface{}) ([]interface{}, bool) {
```

Comment:
It looks like this was moved even though it's staying in the `equinix` package; we should only move things if it's necessary to complete the refactor.

Reply:
I moved the function to it's original place, and not it doesn't show in the diff.